Rain-Drops/index.html

```diff
@@ -0,0 +1,156 @@
+              </p>
+              <h2><span class="points">0</span> points!</h2>
+            </div>
+            <button type="button" class="btn continue">Continue >></button>
+          </div>
+          <div class="game-statistics">
+            <h2>Game Complete</h2>
+            <p>Your Score: <span class="your-score">0</span></p>
+            <div class="scores-user-top">
+              <ol>
+                Your Top Score
+                <li>0</li>
+                <li>0</li>
+                <li>0</li>
+                <li>0</li>
+                <li>0</li>
+              </ol>
+            </div>
+            <button type="button" class="btn play-again">Play Again</button>
+          </div>
+        </div>
+      </div>
+      <div class="management-game">
+        <div class="management-game__score">
+          Score: <span class="counter">0</span>
+        </div>
+        <div class="management-game__enter-keys">
+          <input
+            type="text"
+            class="display"
+            placeholder="0"
+            value="0"
+            disabled
+          />
+          <div class="keys">
+            <button type="button" class="key">7</button>
+            <button type="button" class="key">8</button>
+            <button type="button" class="key">9</button>
+            <button type="button" class="key clear">Clear</button>
+            <button type="button" class="key">4</button>
+            <button type="button" class="key">5</button>
+            <button type="button" class="key">6</button>
+            <button type="button" class="key">1</button>
+            <button type="button" class="key">2</button>
+            <button type="button" class="key">3</button>
+            <button type="button" class="key enter">Enter</button>
+            <button type="button" class="key zero">0</button>
+            <button type="button" class="key delete">Del</button>
+          </div>
+        </div>
+      </div>
+      <div class="menu">
+        <button type="button" class="menu__btn play"><span>Play</span></button>
+        <button type="button" class="menu__btn how-play">
+          <span>How to Play</span>
+        </button>
+        <div class="menu__game-mode">
+          <div class="game-mode__expand">
+            <div class="lines line-one"></div>
+            <div class="lines line-two"></div>
+          </div>
+          <h2 class="game-mode__header">Game Mode</h2>
+          <h3>Operators</h3>
+          <div class="operators">
+            <div
+              class="operators__sign plus active-operator"
+              data-operator="+"
+            ></div>
+            <div class="operators__sign minus" data-operator="-"></div>
+            <div class="operators__sign multiply" data-operator="*"></div>
+            <div class="operators__sign share" data-operator="/"></div>
+          </div>
+          <h3>Range Numbers</h3>
+          <form>
+            <label class="range-numbers">
```

Comment:
One label shouldn't have two inputs inside.  
Apparently, min & max look like text for labels.